conf/default.yml

```diff
@@ -14,6 +14,11 @@ app.contact.tunnel.ssh.host_key_passphrase: REPLACE_WITH_KEY_FILE_PASSPHRASE
 app.contact.tunnel.ssh.socket: 0.0.0.0:8022
 app.contact.tunnel.ssh.user_name: sandcat
 app.contact.tunnel.ssh.user_password: s4ndc4t!
+app.contact.ftp.host: 0.0.0.0
+app.contact.ftp.port: 2222
+app.contact.ftp.pword: caldera
```

Comment:
are you overwriting with something generated if --insecure is not used?

Reply:
Are you asking about default values if someone is using a separate config (e.g. local.yml) and these config variables don't exist?